### PR TITLE
Notices for Jamendo tracks when seeking is impossible

### LINKED ISSUE #680: Jamendo tracks can't be seeked in Blink browsers
## Description

Jamendo does not send an `Accept-Ranges:` header with its audio results, which makes seeking not work in Blink-based browsers like Edge, Chrome, and Opera. This PR adds a notice for these browsers when they attempt to interact with an audio waveform.

We should add a notice when seeking is attempted on one of these tracks.

## Environment

 - Browser: Any blink/chromium based browser

## Resolution

- [x] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by zackkrida**
## Fixes

Fixes #680 by @zackkrida 

## Description

Adds a `featureNotices` feature to the audio track so custom messages can be displayed when any given feature isn't active. It's a bit of an extra abstraction but felt cleaner and simpler than writing some specific hack for this functionality.

## a11y info

When the waveform is disabled, it is no longer focusable, so when users tab from the play button they are sent straight to the next play button. This means they are not explicitly informed about the disabled seek bar, but they're also not incorrectly able to focus the waveform or anything worse like that. I'm not sure how to best approach the _ideal_ solution to this.

## Caveats

I couldn't get our UA detection to run server-only, which I would have liked. Perhaps this could be refactored in the future to add the parsed UA information to a Vuex store instead of injecting it into the Nuxt context, which would preserve the data on client-side renders after the initial server render, but alas, I tried that and wasn't able to get it to work either. For whatever reason it seemed my mutation wouldn't run.

## Testing Instructions

Visit an audio results page in Chrome or Edge and observe the hover effect on a Jamendo track.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VAudioTrack/VWaveform.vue [hunk 1]
```diff
@@ -149,6 +149,14 @@
...
       {{ message }}
     </div>
+
+    <!-- Seek disabled message overlay -->
+    <div
+      v-if="seekDisabledNotice"
+      class="invisible group-hover:visible group-focus:visible absolute w-full inset-0 flex items-center justify-center font-bold text-xsm bg-yellow/75 z-40"
+    >
+      {{ seekDisabledNotice }}
+    </div>
```
Comment (dhruvkb) [inline src/components/VAudioTrack/VWaveform.vue:159]: Imo we use the existing `message` field, with a prop that allows it to be transient. I doubt the two will ever need to overlap.

**Review by dhruvkb — APPROVED** [review]
This is huge good, appreciate the features being popped with notices! Approving with a non-blocking suggestion.

File: src/components/VAudioTrack/VWaveform.vue (hunk 1 above)
Comment (zackkrida) [reply src/components/VAudioTrack/VWaveform.vue:159]: At a minimum they could definitely share some styles; good suggestion.

**Comment by sarayourfriend** [comment]
@zackkrida I like this solution but just letting you know I'm going to take some extra time to review this on using JAWS on a Windows VM to make sure that it's a pleasant experience for screen reader users.

FWIW I think just skipping the seeking _might_ be okay for now? I considered suggesting that we added some kind of extra information into the label on the play/pause button to indicate that seeking would be disabled for that particular audio track, but I want to be wary of the fact that if you narrowed a search to just jamendo tracks then every single play/pause button would include this extra information and could be frustrating for the user.

**Comment by zackkrida** [comment]
> I'm going to take some extra time to review this on using JAWS on a Windows VM 

@sarayourfriend please do, thank you 😄 Keep that VM hot for #662 later too 😆

**Comment by panchovm** [comment]
I added a few mockups on the [Releases](https://www.figma.com/file/w60dl1XPUvSaRncv1Utmnb/Openverse-Releases?node-id=133%3A6869) file showing the alert message above grid results and in the wave area on both [desktop](https://cln.sh/stqwng) and [mobile](https://cln.sh/sNkPG0). The message is primarily static on mobile, yet it shows the playing state through the yellow background.

The `limited to Firefox and Safari` sounds intentional instead of a condition openverse is dealing with.

**Comment by sarayourfriend** [comment]
Oh that's something I hadn't considered, how this will appear to Android Chrome users.

**Pull request merged**